components/ReviewCard.tsx

```diff
@@ -1,28 +1,52 @@
 import { useMutation } from '@apollo/client'
 
-import React, { useState } from 'react'
+import React, { useState, useContext } from 'react'
 import Markdown from 'markdown-to-jsx'
 import dayjs from 'dayjs'
-
+import { GlobalContext } from '../helpers/globalContext'
 import relativeTime from 'dayjs/plugin/relativeTime'
 import ACCEPT_SUBMISSION from '../graphql/queries/acceptSubmission'
 import REJECT_SUBMISSION from '../graphql/queries/rejectSubmission'
-import { Submission } from '../graphql/index'
-
+import { Submission, useAddCommentMutation } from '../graphql/index'
+import { SubmissionComments } from './SubmissionComments'
+import ReviewerProfile from './ReviewerProfile'
 import _ from 'lodash'
+import styles from '../scss/reviewCard.module.scss'
 
 import { Button } from './theme/Button'
 import { Text } from './theme/Text'
 import { MdInput } from './MdInput'
-import ReviewerProfile from './ReviewerProfile'
 import DiffView from './DiffView'
-
+import { updateCache } from '../helpers/updateCache'
 dayjs.extend(relativeTime)
 
 type ReviewCardProps = {
   submissionData: Submission
 }
 
+type CommentType = 'accept' | 'reject' | 'comment'
+
+const RequestChanges: React.FC<{
+  username: string
+  name: string
+  comment: string
+  date?: string
+}> = ({ name, username, comment, date }) => {
+  return (
+    <div className={`${styles.changes} px-2 py-1`}>
+      <div className="d-flex align-items-center">
+        <img src="/assets/requestChanges.svg" className={styles.icon} />
+        <ReviewerProfile name={name} username={username} />
+        <div className={`${styles.font} ml-2`}>
+          requested changes on {dayjs(date).format('dddd, MMMM D, YYYY')}:
```

Comment:
![image](https://user-images.githubusercontent.com/20666236/122102431-0952b000-ce33-11eb-8def-088b0642e730.png)

You'll have to pass unix timestamps as number to get them parsed correctly.